src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.UnitTests/System/Runtime/InteropServices/JavaScript/JavaScriptTestHelper.cs

```diff
@@ -1002,6 +1005,15 @@ public static async Task InitializeAsync()
                 // Log("JavaScriptTestHelper.mjs imported");
             }
         }
+
+        public static Task DisposeAsync()
+        {
+            _module.Dispose();
+            _module = null;
+            // you can set verbose: true to see which proxies are left to the GC to collect
+            ForceDisposeProxies(false, verbose: false);
```

Comment:
Do we want to propagate the `verbose` parameter somehow, probably from some JS configuration? In other words, would it be useful to have the opportunity to switch it to true (without rebuilding runtime)?

Reply:
In this place it's just internal unit test helper.
There is internal `.withInteropCleanupOnExit()` and `interopCleanupOnExit` config options.
When you set it, it will make dump to console on the end of the app.